src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -3,30 +3,74 @@
 import core.basesyntax.dao.CommentDao;
 import core.basesyntax.model.Comment;
 import java.util.List;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
 
 public class CommentDaoImpl extends AbstractDao implements CommentDao {
     public CommentDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
-    public Comment create(Comment entity) {
-        return null;
+    public Comment create(Comment comment) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(comment);
+            transaction.commit();
+            return comment;
+        } catch (HibernateException e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't save comment to DB: " + comment, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(Comment.class, id);
+        } catch (HibernateException e) {
+            throw new RuntimeException("Can't get comment from DB by id: " + id, e);
+        }
     }
 
     @Override
     public List<Comment> getAll() {
-        return null;
+        try (Session session = factory.openSession()) {
+            Query<Comment> getAllCommentsQuery = session.createQuery("from Comment", Comment.class);
+            return getAllCommentsQuery.getResultList();
+        }
     }
 
     @Override
     public void remove(Comment entity) {
-
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.remove(entity);
```

Comment:
here you're using remove and in MessageDao you're using delete, let's choose one of those methods and use everywhere, fix in all places